evalml/pipelines/components/transformers/samplers/undersampler.py

```diff
@@ -13,16 +13,16 @@ class Undersampler(BaseSampler):
     """Random undersampler component. This component is only run during training and not during predict."""
     name = "Undersampler"
     hyperparameter_ranges = {
-        'balanced_ratio': Real(1, 10),
+        'balanced_ratio': Real(1e-10, 1),
```

Comment:
Should `balanced_ratio` be a tunable hyperparameter? If a user passes in `balanced_ratio=0.2` as the argument, shouldn't we keep that ratio for all pipelines? 

Additionally, this equivalent parameter in the oversampler components won't be able to be tuned (due to the additional complexities it requires for multiclass samples). Intuitively, I want to keep the performances of these two component classes similar as well.

Reply:
Issue tracking removal of hyperparameters [here](https://github.com/alteryx/evalml/issues/2104)